bin/server.js

```diff
@@ -43,9 +45,14 @@ app.use(expressWinston.logger({
   baseMeta: {source: 'request', type: 'request'}
 }));
 
+if (!!global.Config.get('vault:token')){
+  global.Config.set('flag:unseal', true);
```

Comment:
Do you want to include a test for this case, where the default `flag:unseal` value changes based on if there's a Vault token or not?